.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+To help us understand and resolve your issue please check that you have provided
```

Comment:
To make it less intimidating I would suggest changing this to something like `**please check that you have provided as much of the information below as possible**  Not all users may be confident with opening a terminal